src/test/java/com/mastercard/developer/signers/SpringHttpRequestSignerTest.java

```diff
@@ -0,0 +1,94 @@
+package com.mastercard.developer.signers;
+
+import com.mastercard.developer.test.TestUtils;
+
+import org.springframework.http.HttpMessage;
+import org.springframework.http.HttpRequest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+
+import java.net.URI;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.security.PrivateKey;
+
+public class SpringHttpRequestSignerTest {
+    
+    private static final HttpMethod method = HttpMethod.POST;
+    private static final String body = "{\"foo\":\"bar\"}";
+    private static final String consumerKey = "Some key";
+    
+    private PrivateKey signingKey;
+	private URI uri;
```

Comment:
Tab! :)

Reply:
Done in the new commits. Replaced tabs with 4 white spaces.